CustomQueries/server/aspnet/RevealSdk.Server/Reveal/DataSourceProvider.cs

```diff
@@ -25,6 +26,22 @@ public class DataSourceProvider : IRVDataSourceProvider
                 }
             }
 
+            if (dataSourceItem is RVMySqlDataSourceItem mySqlDataSourceItem)
+            {
+                var sqlDataSource = (RVMySqlDataSource)mySqlDataSourceItem.DataSource;
+                UpdateDataSource(sqlDataSource);
```

Comment:
This should call the same change data source async method. This needs to be updated for all servers.